clusterloader2/pkg/prometheus/manifests/dashboards/dns.dashboard.py

```diff
@@ -74,18 +74,18 @@
     ),
 ]
 
-COREDNS_PANELS = [
+KUBEDNS_PANELS = [
```

Comment:
I'm not sure if I understand why you need to remove coredns? Can't we have dashboards for both (potentially one of them will be empty depending on the environment we are using)?

Reply:
CoreDNS is not removed, data from CoreDNS will be shown in the same panel as data for kube-dns,.

Actually, we will have single dashboard with kube-dns and CoreDNS data - they both use the same ServiceMonitor job name so we can use the same queries.

If we want to see the difference between them, we can query over 'pods' , but we should not have both CoreDNS and kube-dns running in the same cluster.

I prefer to rename from specific implementation to service name - CoreDNS also uses this service name in it's template.

Reply:
If this panels is intended to view in-cluster DNS data, then I'd rename it to "In-cluster DNS" instead of using "kube-dns". This can be misleading.

Reply:
Actually, we are already on DNS dashboard so it's obvious that we are discussing DNS related values. 'In-cluster' is not verbose enough, at there is no information if which part we are referring to.

Renamed both panels as client/service, because this is the data that is shown.

Left DNS related info in dashboard titles, to increase readability in case of screenshoting graphs.